### ISSUE TITLE
Cron page still links to instructions on drupal.org

### ISSUE THREAD

**Issue by laryn** [backdrop/backdrop-issues#3044]
The cron settings page (**admin/config/system/cron**) still links to [cron documentation on drupal.org](http://drupal.org/cron). That should be moved over to a BackdropCMS site.

![screen shot 2018-04-20 at 12 36 07 pm](https://user-images.githubusercontent.com/871421/39065552-c043572a-4497-11e8-95e5-202b7e99091a.jpg)

It looks like this link is used in a number of places:
https://github.com/backdrop/backdrop/search?utf8=%E2%9C%93&q=%22drupal.org%2Fcron%22&type=

---
PR: https://github.com/backdrop/backdrop/pull/3165

**Comment by jenlampton** [comment]
Do we have a comparable page on backdropcms.org yet? That may be the blocker on this one.  @klonos I think I saw you working on a cron page recently?

**Comment by klonos** [comment]
Sorry for the really late responce guys 😓 ...I may have done some work on the corn settings admin page in the Backdrop UI, but I have not worked on the documentation for it though (there's quite a bit to be copied over from https://www.drupal.org/docs/7/setting-up-cron).

@oadaeh you asked on Gitter re this. I think that the documentation should live in our b.org user guide (as an advanced topic). I made a start (currently unpublished) by copying and adapting the respective overview page from d.org: https://backdropcms.org/user-guide/setting-cron

- I have removed most references to D6
- I have changed "Drupal" to "Backdrop" where appropriate
- I have added a new paragraph (currently in bold - so it can be spotted easily by those that will review/edit before publishing). This refers to the work we have done in #1214, but unfortunately it seems that we have not created a change record for that change 😓 ...so cannot refer people to it from the documentation (another task to be done) ...we also need to mention the new "Use background fetch for cached pages" setting we have introduced in `admin/config/development/performance`.
- The "Disabling "automated cron"" section was mentioning how to manually set the `cron_safe_threshold` variable to 0 in D7. We have changed that to config in Backdrop, so I am mentioning where that setting lives and how to manually change it in the `system.core.json` file. Should we though?
- In D7, there was a way to set `cron_safe_threshold` in settings.php. I have looked through our code, and we don't seem to support that. Should we (separate issue)? If not, should we at least mention that this will not have any effect? Anyway, I have for now left that bit untouched in the unpublished documentation - also in bold, so it can be easily spotted by editors/reviewers.
- D7 offered a way to "lock" `cron_safe_threshold` from being changed via the admin UI. I have filed a separate issue in #3927 to re-introduce that. If we decide not to, then the respective section should be removed from the documentation, or we should mention that this is not supported in Backdrop any more (change record for that too?).

**Comment by klonos** [comment]
...there's just _a lot_ of documentation to be copied across before we can change that link. I will need help with that 😓

**Comment by herbdool** [comment]
I've stripped out some of the text, tweaked it and published it. I think it's better err on the side of keeping it simple and be able to publish it. We can always add more information later. For most people they'll probably use the built-in cron; for the few they can search the web (for now).

**Comment by olafgrabienski** [comment]
@herbdool Great! I've skimmed through https://backdropcms.org/user-guide/setting-cron?no-cache-please. Looks really good to me. In my opinion, we can change the documentation link from d.org to b.org now.

**Comment by herbdool** [comment]
@olafgrabienski want to create a PR for that part?

**Comment by olafgrabienski** [comment]
@herbdool I'd love to, but for at least a week I've no time to do it.

**Comment by olafgrabienski** [comment]
@herbdool Before I create a PR:

Can you change the path of the new "Setting up cron" page from `/user-guide/setting-cron` to just `/setting-cron`? This is to make a change of the URL less likely, see @quicksketch's proposal in a comparable issue: https://github.com/backdrop/backdrop-issues/issues/3560#issuecomment-481450012

Another thing: I've had another look at the page and saw its menu item being out of context of the actual *User guide* menu tree (see screenshot below). Can you change that too?

<img width="494" alt="screen-borg-userguide-cron" src="https://user-images.githubusercontent.com/14188307/68753797-e4b6a680-0605-11ea-93b7-027217c710d2.png">

**Comment by herbdool** [comment]
@olafgrabienski I'm not sure if I've got the authority to override the URL pattern. Though I did change the book the page is in so now it's properly in the User Guide. All the other pages in it contain `/user-guide/` so perhaps we'd have to agree on changing it for most/all of them? Maybe @klonos could weigh in.

**Comment by olafgrabienski** [comment]
> All the other pages in it contain `/user-guide/` so perhaps we'd have to agree on changing it for most/all of them?

Over in https://github.com/backdrop/backdrop-issues/issues/3556#issuecomment-553134592 and below we recently had a similar discussion. IIRC, the policy is to have more simple and pathauto-independent URLs for important documentation sites which are linked from within the Backdrop user interface. For instance, `https://backdropcms.org/user-guide/backing-up-a-site` ended up being on `https://backdropcms.org/backups` (not yet published).

Pinging @jenlampton to clarify the policy.

**Comment by jenlampton** [comment]
> I'm not sure if I've got the authority to override the URL pattern.

We don't want to change the automatic pattern, just the path for any page that is linked from the code, or otherwise deemed "important". This was a request from @quicksketch who really appreciated the URLs for important pages on drupal.org being short and memorable. This stands the test of time, and if we need to "move" the page in the future, we can use our redirects to do so :)

**Comment by herbdool** [comment]
PR ready for review https://github.com/backdrop/backdrop/pull/3165

**Comment by indigoxela** [comment]
Another link to drupal.org replaced. It seems that ZenCI totally failed, but that's for sure unrelated - so RTBC.

**Comment by olafgrabienski** [comment]
@herbdool Looks good but did you miss one file? The PR changes these files:

- core/modules/system/system.admin.inc
- core/modules/system/system.install

The [search link](https://github.com/backdrop/backdrop/search?utf8=%E2%9C%93&q=%22drupal.org%2Fcron%22&type=) in the issue description shows one more:

- core/modules/system/system.api.php

**Comment by herbdool** [comment]
@olafgrabienski thanks, I'll update

**Comment by herbdool** [comment]
Updated.

**Comment by quicksketch** [comment]
Thanks! Merged https://github.com/backdrop/backdrop/pull/3165 into 1.x and 1.16.x.
